include/bayes_tracking/multitracker.h

```diff
@@ -154,19 +155,23 @@ class MultiTracker {
   /**
    * Perform data association and update step for all the current filters, create new ones and remove those which are no more necessary
    * @param om Observation model
+   * @param om_flag Observation model flag (CARTESIAN or POLAR)
    * @param alg Data association algorithm (NN, NN_LABELED, NNJPDA, NN_LABELED)
+   * @param seqSize Minimum number of observations necessary for new track creation
+   * @param seqTime Minimum interval between observations for new track creation
+   * @param stdLimit Upper limit for the standard deviation of the estimated position
    */
   template<class ObservationModelType>
-  void process(ObservationModelType& om, association_t alg = NN, unsigned int seqSize = 5, double seqTime = 0.2)
+  void process(ObservationModelType& om, observ_model_t om_flag = CARTESIAN, association_t alg = NN, unsigned int seqSize = 5, double seqTime = 0.2, double stdLimit = 1.0)
```

Comment:
In order to be upstream compatible, it would be better to add optional arguments at the end, so it doesn't break the interface?